Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:14-alpine
+
+WORKDIR /usr/src/app
+
+COPY package.json ./
+COPY yarn.lock ./
+
+RUN yarn
+
+COPY . .
+
+RUN apk add  --no-cache ffmpeg
+
+CMD [ "npm", "start" ]
```

Comment:
Should we have it run kafka too? or were you thinking kafka would be loaded in a separate container and docker-composed together?

Reply:
Oh, separate container! Docker Composed together.

Reply:
Modifying Docker Compose is step 3 in my multi-step plan.